spire/templates/apps/dovetail-router.yml

```diff
@@ -739,6 +741,25 @@ Resources:
             Effect: Allow
             Principal:
               Service: ecs-tasks.amazonaws.com
+          - Action:
```

Comment:
This statement needs to go under `Policies`, not the `AssumePolicyDocument`. See https://github.com/PRX/Infrastructure/blob/main/spire/templates/apps/feeder.yml#L599 for an example